src/ProjectSystem/Microsoft.VisualStudio.Editors/project.json

```diff
@@ -3,5 +3,9 @@
     "net46": {}
   },
   "dependencies": {
+    "Microsoft.VisualStudio.LanguageServices": {
+      "version": "1.2.0",
+      "suppressParent": "all"
```

Comment:
No, suppressParent prevents the package from being inherited by downstream projects. None of my core projects copy NuGet packages (controlled by CopyNuGetImplementations in the project itself). 'suppressParent' is to prevent DeployTestDependencies from seeing the packages and copying them.